packages/dd-trace/src/plugin_manager.js

```diff
@@ -6,6 +6,15 @@ const plugins = require('./plugins')
 const log = require('./log')
 const { getEnvironmentVariable } = require('../../dd-trace/src/config-helper')
 
+// Test optimization plugins that should only be enabled when isCiVisibility is true
+const TEST_OPTIMIZATION_PLUGINS = new Set([
+  'jest',
+  'vitest',
+  'cucumber',
+  'mocha',
+  'playwright'
```

Comment:
I suppose Cypress is not in this list, because we instrument it through a Cypress plugin and not a tracer plugin?

Reply:
yes that's correct